### PR TITLE
chore: fix discord link in footer

### PR DESCRIPTION
**Pull request by BlankParticle**

### CODE REVIEW

File: app.config.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default defineAppConfig({
     links: [
       {
         icon: "i-simple-icons-discord",
-        to: "https://discord.postalserrver.com",
+        to: "https://discord.postalserrver.io",
```
Comment (willpower232) [inline app.config.ts:49]:
```suggestion
        to: "https://discord.postalserver.io",
```

**Review by willpower232 — CHANGES_REQUESTED** [review]

**Review by willpower232 — APPROVED** [review]

**Comment by willpower232** [comment]
Thanks!

File: app.config.ts (hunk 1 above)
Comment (BlankParticle) [reply app.config.ts]: lol I didn't fix the other typo 😅

Comment (willpower232) [reply app.config.ts]: happens to us all :wink:

**Pull request merged**